nbsphinx_link/__init__.py

```diff
@@ -140,6 +141,9 @@ def collect_extra_media(extra_media, source_file, nb_path, document):
         dest_path = utils.relative_path(nb_path, src_path)
         dest_path = os.path.normpath(os.path.join(source_dir, dest_path))
         if os.path.exists(src_path):
+            if not any_dirs and os.path.isdir(src_path):
+                any_dirs = True
+                document.settings.env.note_reread()
```

Comment:
I think ``document.settings.env.note_reread()`` should be called at the end of ``collect_extra_media``.

```python
if  any_dirs :
    document.settings.env.note_reread()
```
Else ``any_dirs `` is pointless.

Reply:
`any_dirs` is only there to shortcut the if condition (i.e. prevent `isdir` from being called). Maybe the simpler version reads better, and the performance gain is negligible. I'll change it.

Reply:
Pushed

Reply:
Ups my bad, never mind (should get another coffee) >.<